public/app/features/plugins/admin/components/PluginList.tsx

```diff
@@ -18,8 +18,8 @@ interface Props {
 export const PluginList = ({ plugins, displayMode }: Props) => {
   const isList = displayMode === PluginListDisplayMode.List;
   const styles = useStyles2(getStyles);
-  const location = useLocation();
-  const pathName = config.appSubUrl + location.pathname;
+  const { pathname } = useLocation();
+  const pathName = config.appSubUrl + (pathname.endsWith('/') ? pathname.slice(0, -1) : pathname);
```

Comment:
Although I think probably the best would be if the last item in the chain (PluginListItem) would do the validation - so whatever gets passed in it renders the correct links -, this change does the job as well. 👍